### DISCUSSION TITLE
Page Header Tags issue when no HTML element is used

### DISCUSSION THREAD

**Discussion by Timo-Breumelhof** [dnnsoftware/Dnn.Platform#3148] [Ideas]
## Description of Problem
I recently had one page on a site that had a strange extra distance somewhere in the page. (fixed header offset actually). After a long search I found that the admin had added tags to the "Page Header Tags".
But only text noting else, apparently he misunderstood the term "tags". 
This meant that plain text was injected in the head, which lead to the rendering issue.

## Proposed Solution Option 1
I suggest to add a regex validator to the input filed to make sure anything inserted is valid.?

## Proposed Solution Option 2 (If Needed, repeat for more)
Strip out all content that's not inside an html element on render?

## Alternatives Researched
Documentation of any other items researched for resolution.

## Affected version

* [ ] 10.0.0 nightly build
* [x] 9.4.1 nightly build
* [x] 9.4.0 latest supported release

**Comment by stale** [comment]
We have detected this issue has not had any activity during the last 90  days. That could mean this issue is no longer relevant and/or nobody has  found the necessary time to address the issue. We are trying to keep the  list of open issues limited to those issues that are relevant to the  majority and to close the ones that have become 'stale' (inactive). If no  further activity is detected within the next 14 days, the issue will be  closed automatically.
If new comments are are posted and/or a solution (pull request) is  submitted for review that references this issue, the issue will not be  closed. Closed issues can be reopened at any time in the future. Please remember those participating in this open source project are volunteers  trying to help others and creating a better DNN Platform for all. Thank  you for your continued involvement and contributions!

**Comment by Timo-Breumelhof** [comment]
...

**Comment by valadas** [comment]
To prevent stalebot to keep closing this since nobody picked up this feature-request yet, I am moving this to the feature-request discussions.

Thanks.
